Assets/Mirage/Runtime/InterestManagement/InterestManager.cs

```diff
@@ -0,0 +1,286 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using Mirage.Logging;
+using Mirage.Serialization;
+using UnityEngine;
+
+namespace Mirage.InterestManagement
+{
+    public class InterestManager
+    {
+        static readonly ILogger logger = LogFactory.GetLogger(typeof(InterestManager));
+
+        #region Fields
+
+        protected internal readonly ServerObjectManager ServerObjectManager;
+        private ObserverData[] _visibilitySystems;
+        private readonly int _initialSystems;
+        private readonly List<ObserverData> _observerSystems = new List<ObserverData>();
```

Comment:
This seems to be un-used

Reply:
It is used by the tank script in one of the examples i will remove it and try update the script better.